src/web/components/Home/RotateAPIKeysCard.tsx

```diff
@@ -0,0 +1,47 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Link } from 'react-router-dom';
+
+import { ApiKeyDTO } from '../../../api/services/adminServiceHelpers';
+import {
+  deduplicateStrings,
+  formatStringsWithSeparator,
+  formatUnixDate,
+} from '../../utils/textHelpers';
+
+import './RotateAPIKeysCard.scss';
+
+type RotateApiKeysCardProps = Readonly<{
+  apiKeysToRotate: ApiKeyDTO[];
+}>;
+
+function RotateApiKeysCard({ apiKeysToRotate }: RotateApiKeysCardProps) {
+  const getDatesCreated = () => {
+    const datesCreated = apiKeysToRotate.map((apiKey) => formatUnixDate(apiKey.created).toString());
+    const datesCreatedString = formatStringsWithSeparator(deduplicateStrings(datesCreated));
+    return datesCreatedString;
+  };
+
+  return (
+    <div className='rotate-api-keys-card'>
+      <FontAwesomeIcon icon='triangle-exclamation' className='warning-button' />
```

Comment:
We already have `exclamation-triangle` 😂 Let's use that instead. Then we can remove the changes in `configureFontAwesomeLibrary.ts`

Reply:
Ooh yes I did see that - I was just confused because on the FontAwesome webiste, the icon shows `triangle-exclamation`.  I agree we should only have one, but should I change the ones that use `exclamation-triangle` to `triangle-exclamation`.  I'm not sure how they both work, but maybe the order doesn't matter? 
![Uploading TriangleExclamation.PNG…]()

Reply:
yeah good point. I now see that [exclamation-triangle](https://fontawesome.com/v5/icons/exclamation-triangle?f=classic&s=solid) is version 5 and [triangle-exclamation](https://fontawesome.com/icons/exclamation-triangle?f=classic&s=solid) is version 6. We may as well use the latest version, so I'd suggest changing all of them to `triangle-exclamation` and removing `exclamation-triangle` :)